src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -680,6 +1057,11 @@ public override IOperation VisitCoalesce(ICoalesceOperation operation, int? capt
             return new FlowCaptureReference(resultCaptureId, operation.Syntax, operation.Type, operation.ConstantValue);
         }
 
+        private static BasicBlock.Branch RegularBranch(BasicBlock destination)
```

Comment:
Can you add a prototype comment for organizing this file before we merge? We should have our common private static helpers in a common place, currently we seem to have a bunch in the middle of the file between visit methods.

Reply:
> We should have our common private static helpers in a common place

There is no such requirement.

---
In reply to: [167692787](https://github.com/dotnet/roslyn/pull/24770#discussion_r167692787) [](ancestors = 167692787)

Reply:
It helps with readability of the file. I've been wanting to clean up our CSharp/VB operation builders as well, they're a mess.

---
In reply to: [167696142](https://github.com/dotnet/roslyn/pull/24770#discussion_r167696142) [](ancestors = 167696142,167692787)

Reply:
These are your personal preferences and I am sure not everyone on the team would agree with you point of view

---
In reply to: [167696873](https://github.com/dotnet/roslyn/pull/24770#discussion_r167696873) [](ancestors = 167696873,167696142,167692787)